api/tests/test-helper.js

```diff
@@ -115,6 +115,17 @@ async function insertUserWithRoleCertif() {
   return user;
 }
 
+function buildParameterizedTests(params, title, callee) {
+  describe('parameterized test', function () {
+    // eslint-disable-next-line mocha/no-setup-in-describe
+    params.forEach((test) => {
+      it(title(test), function () {
+        return callee(test);
+      });
+    });
+  });
+}
```

Comment:
`title` semble statique alors que c'est une fonction. On pourrait la renomment plus explicitement `generateTitle`
```suggestion
function buildParameterizedTests(params, generateTitle, callee) {
  describe('parameterized test', function () {
    // eslint-disable-next-line mocha/no-setup-in-describe
    params.forEach((test) => {
      const title = generateTitle(test);
      it(title, () => callee(test));
    });
  });
}
```